test/built-ins/Array/prototype/concat/is-concat-spreadable-val-falsey.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 22.1.3.1
+description: >
+    The `Symbol.isConcatSpreadable` property is defined and coerces to `false`
+info: >
+    1. Let O be ToObject(this value).
+    2. ReturnIfAbrupt(O).
+    3. Let A be ArraySpeciesCreate(O, 0).
+    4. ReturnIfAbrupt(A).
+    5. Let n be 0.
+    6. Let items be a List whose first element is O and whose subsequent
+       elements are, in left to right order, the arguments that were passed to
+       this function invocation.
+    7. Repeat, while items is not empty
+      a. Remove the first element from items and let E be the value of the element.
+      b. Let spreadable be IsConcatSpreadable(E).
+      c. ReturnIfAbrupt(spreadable).
+      d. If spreadable is true, then
+         [...]
+      e. Else E is added as a single item rather than spread,
+         [...]
+
+    ES6 22.1.3.1.1: Runtime Semantics: IsConcatSpreadable ( O )
+
+    1. If Type(O) is not Object, return false.
+    2. Let spreadable be Get(O, @@isConcatSpreadable).
+    3. ReturnIfAbrupt(spreadable).
+    4. If spreadable is not undefined, return ToBoolean(spreadable).
+features: [Symbol.isConcatSpreadable]
+---*/
+
+var item = [1];
```

Comment:
could you change this to var item = [1,2];

First let me state that this test case is correct, b\c of the array ref compares, but I think it needs to be more explicitly clearing what is going on here. if the array you are adding/concating has length 2 and the length of the concated array is length 1 it is more abundantly clear that a spread did not happen.